src/js/components/DataFilters/__tests__/__snapshots__/DataFilters-test.tsx.snap

```diff
@@ -848,7 +848,7 @@ exports[`DataFilters drop badge 1`] = `
   flex-direction: column;
   flex: 0 0 auto;
   justify-content: center;
-  border-radius: 24px;
+  border-radius: 12px;
```

Comment:
round={true} will resolve to "12px" now. Expected change based on updated t-shirt ranges.